src/slack.js

```diff
@@ -268,7 +288,9 @@ async function handleOrderAction(event) {
       actions: attachment.actions.filter((action) => action.name !== 'subsidy'),
     }]})
 
-    await addSubsidy(orderId)
+    const {order, items} = await getOrderAndItemsFromDb(orderId)
```

Comment:
These variables are used in 3 `if` clauses, move them above the `if` so they can be reused.

Reply:
Moved up and I added `shopId`, `count` and `url` to the query so I could reuse items in all the `if`s.